rest-api/tools/import_organizations.py

```diff
@@ -190,13 +192,17 @@ def _entity_from_row(self, row):
     phone = row.get(SITE_PHONE_COLUMN)
     admin_email_addresses = row.get(SITE_ADMIN_EMAIL_ADDRESSES_COLUMN)
     link = row.get(SITE_LINK_COLUMN)
+    digital_scheduling_status = DigitalSchedulingStatus(row[DIGITAL_SCHEDULING_STATUS].upper())
```

Comment:
Has this column been added to the sheet and populated yet? It's not reflected in data/sites.csv...

This will fail until it is...

Once it is, we may want to add DIGITAL_SCHEDULING_STATUS to the array of required columns on line 125.

Reply:
yes I altered the csv to reflect. Adding to required now.